packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/DiagramRenderer.tsx

```diff
@@ -55,17 +59,11 @@ import { useDiagramSelection } from './selection/useDiagramSelection';
 import { useSnapToGrid } from './snap-to-grid/useSnapToGrid';
 
 import 'reactflow/dist/style.css';
-import { NodeContext } from './node/NodeContext';
-import { NodeContextValue } from './node/NodeContext.types';
 
 const GRID_STEP: number = 10;
 
-export const DiagramRenderer = ({
-  diagramRefreshedEventPayload,
-  diagramDescription,
-  selection,
-  setSelection,
-}: DiagramRendererProps) => {
+export const DiagramRenderer = ({ diagramRefreshedEventPayload, selection, setSelection }: DiagramRendererProps) => {
+  const { diagramDescription } = useContext<DiagramDescriptionContextValue>(DiagramDescriptionContext);
```

Comment:
We could have a custom hook for that, it feels better than raw context since it would also give us a place to store some functions related to the diagram descriptions